tests/unit/test_connection.py

```diff
@@ -42,13 +41,24 @@ def __init__(self, responses):
         {'request-id': 2},
         {'request-id': 3},
     ]
-    con._get_sll = mock.MagicMock()
+    minimal_facades = [{'name':'Pinger','versions':[1]}]
```

Comment:
Are those files parsed  by lint? This dict here seems to miss the usual spaces after ":" and ",".

Reply:
tox excludes the test files (deliberate choice, I think). I'll run lint on the test files too.

Reply:
I think the tests might be excluded.  No objection to including them, though.

Reply:
Included in #188